src/cmdc_tools/datasets/__init__.py

```diff
@@ -1,17 +1,12 @@
-from .base import DatasetBaseNoDate, InsertWithTempTable, DatasetBaseNeedsDate
-from .db_util import fast_to_sql, TempTable
-
+from .base import DatasetBaseNeedsDate, DatasetBaseNoDate, InsertWithTempTable
 from .bea import CountyGDP
-from .cex import DailyStateLex, DailyCountyLex, CountyDex, StateDex
-from .covidactnow import (
-    CANCountyActuals,
-    CANCountyTimeseries,
-    CANStateActuals,
-    CANStateTimeseries,
-)
+from .cex import CountyDex, DailyCountyLex, DailyStateLex, StateDex
+from .covidactnow import (CANCountyActuals, CANCountyTimeseries,
+                          CANStateActuals, CANStateTimeseries)
 from .covidtrackingproject import CTP
+from .db_util import TempTable, fast_to_sql
 from .dol import StateUIClaims
+from .jhu import DailyReports, DailyReportsUS, Locations
+from .official import LA, Alaska, CACountyData, Imperial, Maryland, SanDiego
```

Comment:
I think this needs to import `Arkansas` and `MA` as well

Reply:
Good catch -- Now they're imported.